File: bindings/java-ffm/src/main/java/org/maplibre/nativeffi/log/LogRecord.java
package org.maplibre.nativeffi.log;

/** Copied Maplibre Native log record delivered to a Java log callback. */
public record LogRecord(
    LogSeverity severity,
    int rawSeverity,
    LogEvent event,
    int rawEvent,
    long code,
    String message) {}

File: bindings/kotlin-native/src/nativeMain/kotlin/org/maplibre/nativeffi/log/LogRecord.kt
package org.maplibre.nativeffi.log

/** Copied Maplibre Native log record delivered to a Kotlin log callback. */
public data class LogRecord(
  public val severity: LogSeverity,
  public val rawSeverity: Int,
  public val event: LogEvent,
  public val rawEvent: Int,
  public val code: Long,
  public val message: String,
)
